README.md

```diff
@@ -114,6 +114,10 @@ class CustomTokenProvider(TokenProvider):
 
 
 sdk = SDK(token_provider=CustomTokenProvider(), group_id=group_id)
+
+""" Alternatively, create a custom TokenProvider that inherits from ExpiringTokenProvider. You should define a 
+    custom _query_token method which fetches your token. See Auth0TokenProvider implementation for an example.
+"""
```

Comment:
I tried to think of a way to indicate when you'll want to use this. But I guess you just have to know. So at least being reminded it exists is probably sufficient